### DISCUSSION TITLE
Allow Trend Functionality in Charts

### DISCUSSION THREAD

**Discussion by jaserodley** [astashov/liftosaur#242] [Ideas]
It would be great if graphs can show more of a trend instead of individual workouts.

Following JnT 2.0, with volume changing daily and weight progression not always linear, the day to day charts are very volatile. Zoom out and you can see a bit of a trend, but it's not very clear.

I'd like to be able to see calculated 1RM charts that group recent data points, or smooth out the lines, so they show a clearer trend line instead of individual workouts.

This will help users to see trajectory over the last 12 months or, as the app matures, multiple years.

Replicating the chart system in the Withings app could be an option. Alternatively, there could be a setting in the charts section along the lines of "Show 1RM Trendline".

**Comment by jaserodley** [comment]
I see we got a few upvotes for this. Is there any chance we can get this integrated @astashov? I think it can help us to power through plateaus.

**Comment by joner391** [comment]
This is probably one of the nicest things about the training app ive been using before Liftosaur. Having a few options on how to look at historic data and get an instant overview is quite motivating.
